src/main/java/core/basesyntax/service/strategy/impl/BuyOperationHandler.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service.strategy.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.strategy.OperationHandler;
+
+public class BuyOperationHandler implements OperationHandler {
+
+    @Override
+    public void updateNumberOfFruit(FruitTransaction fruitTransaction) {
+        Integer numberAfterSelling = Storage.STORAGE.get(fruitTransaction.getFruit())

```

Comment:
it's better to use inner variables to make your code readable for Storage.STORAGE.get(fruitTransaction.getFruit()) and fruitTransaction.getQuantity()